src/components/dialogs/parameters/shortcircuit/short-circuit-parameters-utils.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { IntlShape } from 'react-intl';
+import React, { useMemo } from 'react';
+import { STATUS } from '../../../utils/constants';
+import { Lens } from '@mui/icons-material';
+
+export const useIntlPredefinedParametersOptions = (intl: IntlShape) => {
```

Comment:
can you remove useMemo from here, and use it on the parent importing this function `components/dialogs/parameters/shortcircuit/short-circuit-parameters.tsx`